openlibrary/accounts/model.py

```diff
@@ -669,13 +673,13 @@ def create(
                 return ia_account
 
             elif 'screenname' not in response.get('values', {}):
-                errors = '_'.join(response.get('values', {}))
-                raise ValueError(errors)
+                raise OLAuthenticationError('undefined_error')
```

Comment:
Created new `undefined_error` for this case.  I am not familiar with the types of errors that can occur here, I was unable to find any existing errors in Sentry for insight.